create_infotexts.py

```diff
@@ -360,33 +365,34 @@ def process_depicted_place(self, place_string, places_mapping):
             return
 
         if place_string in places_mapping:
-            if not places_mapping[place_string]["wikidata"] == "-":
-                 place_as_wikitext = "{{city|1={wikidata}}}".format(
-                    wikidata=places_mapping[place_string]["wikidata"][2:]
-                )
-            elif place_string == "Stockholm":  # Mainly interiors from buildings gardens
-                self.meta_cats.append("[[Media_contributed_by_SMVK_taken_in_Stockholm]]")
-                self.data["depicted_place"] = ""
+            if place_string == "Stockholm":  # Mainly interiors from buildings gardens
+                # print("Stockholm in places_mapping{}".format(place_string))
+                self.meta_cats.append("Media_contributed_by_SMVK_without_mapped_place_value")
```

Comment:
I thought it would be overkill, but since it can be used to send to Stockholmiana people I see the point of keeping it. Added "Media_contributed_by_SMVK_taken_somewhere_in_Stockholm" 
done